TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -323,6 +323,14 @@ public class TrafficManagerTool
                     e.Value.Cleanup();
                 }
             }
+
+            // Disabled camera may indicate the main camera change.
+            // Reinitialize camera cache to make sure we will use the correct one
+            if (!InGameUtil.Instance.CachedMainCamera.enabled) {
+                Log.Info("CachedMainCamera disabled - camera cache reinitialization");
```

Comment:
why log?

Reply:
Why not, it's internal info, at worst it will be single occurrence, but may suggest possible place necessary to check when the user report issues with raycasiting or other screen resolution/main camera dependent features

Reply:
if you assume it will help you debug user issue, kk.